package.json

```diff
@@ -19,40 +19,43 @@
   },
   "scripts": {
     "release": "changeset publish",
-    "test": "eslint . && eslint --config base.js .",
+    "test": "eslint --config index.js . && eslint --config base.js .",
     "changeset-version": "changeset version && prettier --write ."
   },
   "homepage": "https://github.com/seek-oss/eslint-config-seek#readme",
   "dependencies": {
-    "@babel/core": "^7.22.6",
-    "@babel/eslint-parser": "^7.22.6",
-    "@babel/preset-react": "^7.22.5",
-    "eslint-plugin-cypress": "^3.0.0",
-    "@typescript-eslint/eslint-plugin": "^7.2.0",
-    "@typescript-eslint/parser": "^7.2.0",
-    "eslint-config-prettier": "^8.8.0",
-    "eslint-import-resolver-typescript": "^3.6.1",
-    "eslint-plugin-import": "^2.27.5",
-    "eslint-plugin-jest": "^27.9.0",
-    "eslint-plugin-react": "^7.32.2",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "find-root": "^1.1.0"
+    "@babel/core": "^7.25.2",
+    "@babel/eslint-parser": "^7.25.1",
```

Comment:
I think we don't need any babel stuff anymore. `@typescript-eslint/parser` handles Typescript and TSX for us, and eslint supports JSX out of the box.

From what I've seen in other eslint configs, this should work. The thing we lose by dropping babel is support for parsing experimental syntax. But we still get [everything that's at stage 4 in the spec](https://github.com/eslint/eslint/?tab=readme-ov-file#what-ecmascript-versions-does-eslint-support), which should be fine for most, if not all, use cases.